src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -9,11 +9,10 @@ public boolean isOddNumber(int a) {
 		int tmp = (a-1)%2;
 		int foo = 0;
 		
-		if (tmp != 0) {
+		if (tmp < 0) { 
 			return true;
 		}
 		return false;
```

Comment:
This line (15) has been identified with a suspiciousness value of 28.57%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_3.NopolExampleTest#test3`
- `nopol_examples.nopol_example_3.NopolExampleTest#test1`
- `nopol_examples.nopol_example_3.NopolExampleTest#test2`
- `nopol_examples.nopol_example_3.NopolExampleTest#test6`
</details>